### DISCUSSION TITLE
part-ii/2021-7/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#365] [Question comments]
Part II 2021 Algebraic Topology: Paper 4, Section II, 21F 

https://questions.tripos.org/part-ii/2021-7/

**Comment by artie2000** [comment]
I wasn't able to do this fully rigorously, but the idea is to take a triangulation with each branch point as a vertex, subdivide so that all simplies are evenly covered (except possibly at a branch vertex), and lift the triangulation to $\Sigma_g$ by tracking the two lifts of each simplex.

It's the same idea as ES4Q4 (2021) and the proof of Riemann-Hurwitz in Riemann Surfaces.

The formula is $n=2g-2$.
